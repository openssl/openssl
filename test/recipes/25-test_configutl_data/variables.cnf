# vim:ft=conf
openssl_conf = openssl_init

default_var = ABC
nested = "\${default_var}"

[othersection]

[test]
# These should expand to ABC read from the default section
0.recipient = ${default_var}
1.recipient = $default_var
2.recipient = $(default_var)
# These should expand to DEF as the other section was explicitly referenced
3.recipient = ${othersection::default_var}
4.recipient = $othersection::default_var
5.recipient = $(othersection::default_var)

[test2]
default_var = GHI
# These should expand to GHI since the local section is always searched first 
0.recipient = ${default_var}
1.recipient = $default_var

[test3]
.pragma dollarid:on
# Out of these, the first should contain the literal "$default_var", the others should expand
0.recipient = literal$default_var
1.recipient = expanded${default_var}
2.recipient = expanded$(default_var)

[test4]
recipient = literal$nested
