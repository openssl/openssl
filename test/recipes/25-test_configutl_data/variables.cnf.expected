openssl_conf = openssl_init
default_var = ABC
nested = \${default_var}

[othersection]

[test]
0.recipient = ABC
1.recipient = ABC
2.recipient = ABC
3.recipient = ABC
4.recipient = ABC
5.recipient = ABC

[test2]
default_var = GHI
0.recipient = GHI
1.recipient = GHI

[test3]
0.recipient = literal\$default_var
1.recipient = expandedABC
2.recipient = expandedABC

[test4]
recipient = literal\$nested
